docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+---
+layout: vNext
+title:  "Setting up and Running a Performance Benchmark on an existing network"
+categories: 
+permalink: /vNext/fabric-tutorial/tutorials-fabric-existing/
+order: 
+---
+## Table of Contents
+{:.no_toc}
+
+- TOC
+{:toc}
+
+## Overview
+This tutorial takes you through performance testing a smart contract on a pre-existing Fabric network using Caliper. 
+
+This tutorial follows from the test-network [fabric tutorial](https://hlf.readthedocs.io/en/latest/test_network.html). You need to have installed and instantiated the `asset-transfer-basic` smart contract (part two of the Fabric tutorial).
```

Comment:
I'd add a quick command summary for getting the Fabric network to the required state. This way we can be sure that the network is set up correctly by the user. 
Strictly with the following disclaimer:
> The below command list is a minimalist quick step guide to get the Fabric network up and running. __To understand and troubleshoot them, refer to the Fabric documentation!__

```bash
# clone a fixed version of the samples repo
git clone https://github.com/hyperledger/fabric-samples.git
cd fabric-samples
git checkout v2.1.1
# install the Fabric tools and add them to PATH
curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.0 1.4.8 -s
export PATH=$PATH:$(pwd)/bin
# create and initialize the network
cd test-network
./network.sh up createChannel
./network.sh deployCC -l javascript
```

Reply:
added the code block 🦖